sos/report/plugins/ssh.py

```diff
@@ -18,6 +18,12 @@ class Ssh(Plugin, IndependentPlugin):
     plugin_name = 'ssh'
     profiles = ('services', 'security', 'system', 'identity')
 
+    option_list = [
+        PluginOpt('userconfs', default='True', val_type=str,
```

Comment:
`default` should be a boolean here, not a string. We do string conversion to bools from the CLI, but internally we should still be using the correct native type.

Reply:
fixed issue!